### DISCUSSION TITLE
export type * from "@maplibre/maplibre-gl-style-spec"

### DISCUSSION THREAD

**Discussion by AhmedBenJaballah** [maplibre/maplibre-gl-js#3179] [Q&A]
Hello everyone, I recently installed MapLibre GL  3.3.1 and wanted to use it with Angular/TypeScript. When I started the application, I encountered the following error:

Error TS1383: Only named exports may use 'export type'.
export type * from "@maplibre/maplibre-gl-style-spec"

However, I was able to solve the problem by navigating to node_modules/maplibre-gl/dist/maplibre-gl.d.ts and then deleting the word 'type' from the line 11564  'export type * from "@maplibre/maplibre-gl-style-spec".'

Does anyone have another solution to solve this problem?

**Comment by AhmedBenJaballah** [comment]
the solution was just adding "skipLibCheck": true, in the tsconfig.json

**Comment by neodescis** [comment]
FYI, this isn't needed in the current version of Angular. The problem is that the syntax is only supported in more recent versions of TypeScript.
